pages/blog/create.jsx

```diff
@@ -8,26 +8,42 @@ import Send from '@mui/icons-material/Send';
 import useStore from '../../hooks/useStore';
 import TextField from '@mui/material/TextField';
 import Container from '@mui/material/Container';
+import SendIcon from '@mui/icons-material/Send';
 import Richtext from '../../components/Richtext';
 import Typography from '@mui/material/Typography';
+import Autocomplete from '@mui/material/Autocomplete';
 
```

Comment:
you don't need to import useRouter? :)

Reply:
I need to import useRouter because I'm using it on line 17. If you missed the import, it's on line 5.